client/src/views/users/utils.ts

```diff
@@ -0,0 +1,105 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+import { SmallDisplayCategoryUserContextMenu, SmallDisplayUserContextMenu } from '@/components/small-display';
+import { UserModel } from '@/components/users';
+import { UserProfile } from '@/parsec';
+import UserContextMenu, { UserAction } from '@/views/users/UserContextMenu.vue';
+import { modalController, popoverController } from '@ionic/vue';
+
+async function canUpdateProfile(user: UserModel, users: UserModel[], clientIsAdmin: boolean): Promise<boolean> {
```

Comment:
We could avoid having two parameters and just have `users: Array<UserModel>` and `return users.some((u) => u.currentProfile !== UserProfile.Outsider && !u.isRevoked());`